modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/styles/components/_quick-links.scss

```diff
@@ -0,0 +1,10 @@
+.quick-links-container {
+	min-width: 343px;
+	right: 0;
+	width: 343px;
+}
+
+.link-body {
```

Comment:
Should be placed before .quick-links-container or inside of quick-links-container